content/kit/servo-assembly/GPIO.md

```diff
@@ -0,0 +1,14 @@
+---
+title: GPIO
+---
+
+The arduino allows you to connect your kit to your own electronics. It has 14 digital I/O pins, and 4 analogue. The analogue pins can read an analogue signal from 0 to 5V. The board also has a couple of ground pins, as well as some pins fixed at 3.3V and 5V output.
+
+The pin layout of the servo assembly is the same as an Arduino uno.
+
+![Pin Map](https://raw.githubusercontent.com/Bouni/Arduino-Pinout/master/Arduino%20Uno%20R3%20Pinout.png?width=50pc)
+
+
+{{% notice note %}}
+The servo hat communicates using 2 of the analogue output pins (4 and 5), and so these are inaccessible.
```

Comment:
Please change "analogue output" -> "analogue input".

Also, "inaccessible" -> "reserved". You can still physically plug stuff into them; it'll just make your servo shield behave weirdly.